src/base/perf_monitor.c

```diff
@@ -3358,6 +3359,7 @@ perfmon_destroy_parallel_stats (THREAD_ENTRY * thread_p)
   if (thread_p->m_px_stats != NULL)
     {
       free_and_init (thread_p->m_px_stats);
+      thread_p->m_uses_px_stats = false;
     }
```

Comment:
```suggestion
      thread_p->m_uses_px_stats = false;
    }
  assert (thread_p->m_uses_px_stats == false);
```
assert를 추가하는 것은 어떨까요?

Reply:
https://github.com/CUBRID/cubrid/pull/6442/commits/45a768b7ecc716fcd40549a247c4a9dbbf6438cf 에서 적용했습니다.